hash/amiga_apps.xml

```diff
@@ -3,7 +3,7 @@
 
 <softwarelist name="amiga_apps" description="Commodore Amiga Application disks">
 
-	<software name="appetide">
+	<software name="appetize">
```

Comment:
This was not a typo - it intentionally had the "de" suffix because it's a German release.  I think it's probably worth leaving the country suffix in for the parent in this case whether you go for `appetizede` or `appetide`.

Reply:
I get that this is not a typo. Usually though the parent doesn't have any language or version indicator in it's name. Basically, if you type `sf2` you probably expect to get the parent version of Street Fighter 2, not caring that it is the "World 910522" version.

Reply:
It's a bit different with a software list like this.  Putting aside the rather arbitrary nature of the parent/clone relationship, the parent for arcade games is generally supposed to be the most widespread, English-language release, mainly for general accessibility.  But if you fire up the wrong release of an arcade game, you probably aren't going to be massively inconvenienced

In this case, it's a piece of software that was localised for a number of regions, and we have a German release as the parent.  I think the original practice in this software list of making the release region explicit in parents is useful.